hack/release.sh

```diff
@@ -17,24 +17,18 @@ fi
 if [[ -z "$PRERELEASE" ]]; then
     PRERELEASE=false
 fi
-    
+
 echo "uploading files:"
-ls -1a ./dist/*.tar.gz ./dist/*.sha256
+ls -1a ./dist/*.tar.gz ./dist/*.sha256 ./manifests/runtime.yaml
```

Comment:
why do we suddenly need this?

Reply:
yeah, for a while i thought i'd add a built kustomization manifest to the artifact, so i looked at the code. i just noticed that the ls command is not identical to the actual files being uploaded as artifact. it doesn't really matter